utils/python/CIME/test_status.py

```diff
@@ -18,21 +18,23 @@
 # Special statuses that the overall test can be in
 TEST_DIFF_STATUS     = "DIFF"   # Implies a failure in one of the COMPARE phases
 NAMELIST_FAIL_STATUS = "NLFAIL" # Implies a failure in the NLCOMP phase
+TEST_BFAIL_STATUS    = "BFAIL"  # Implies baselines are missing in the COMPARE_baseline phase
 
 # The valid phases
-INITIAL_PHASE         = "INIT"
-CREATE_NEWCASE_PHASE  = "CREATE_NEWCASE"
-XML_PHASE             = "XML"
-SETUP_PHASE           = "SETUP"
-NAMELIST_PHASE        = "NLCOMP"
-SHAREDLIB_BUILD_PHASE = "SHAREDLIB_BUILD"
-MODEL_BUILD_PHASE     = "MODEL_BUILD"
-RUN_PHASE             = "RUN"
-THROUGHPUT_PHASE      = "TPUTCOMP"
-MEMCOMP_PHASE         = "MEMCOMP"
-MEMLEAK_PHASE         = "MEMLEAK"
-COMPARE_PHASE         = "COMPARE" # This is one special, real phase will be COMPARE_$WHAT
-GENERATE_PHASE        = "GENERATE"
+INITIAL_PHASE          = "INIT"
+CREATE_NEWCASE_PHASE   = "CREATE_NEWCASE"
+XML_PHASE              = "XML"
+SETUP_PHASE            = "SETUP"
+NAMELIST_PHASE         = "NLCOMP"
+SHAREDLIB_BUILD_PHASE  = "SHAREDLIB_BUILD"
+MODEL_BUILD_PHASE      = "MODEL_BUILD"
+RUN_PHASE              = "RUN"
+THROUGHPUT_PHASE       = "TPUTCOMP"
+MEMCOMP_PHASE          = "MEMCOMP"
+MEMLEAK_PHASE          = "MEMLEAK"
+COMPARE_PHASE          = "COMPARE" # This is one special, real phase will be COMPARE_$WHAT
+COMPARE_BASELINE_PHASE = "COMPARE_baseline"
```

Comment:
Right, you could just revert that. However, I personally feel that having a separate COMPARE_BASELINE_PHASE is a step in the right direction. Though feel free to do whatever you prefer here - if you revert it, we can always reintroduce it later to address #396